src/Tmds.Ssh/SftpChannel.cs

```diff
@@ -1101,6 +1108,8 @@ private async ValueTask DownloadFileAsync(string remotePath, string localPath, l
 
         await remoteFile.CloseAsync(cancellationToken).ConfigureAwait(false);
 
+        return;
+
```

Comment:
Can you remove these two lines?

Reply:
that was a ReSharper recommendation when using functions in functions, but i will remove it